assignment.js

```diff
@@ -57,13 +64,21 @@ Use the following test cases to confirm your program meets the success criteria
 2. Given the following:
     problem1Numbers = [9, 18, 62, 3, 17]
 
-   Result: 0
+   Result: 21.8
 
 ************/
 
+
 console.log('Problem 2:')
 
 // Add your code below this line
+let numArray = [9, 18, 62, 3, 17]
+  let sum = 0
+  numArray.forEach(function(numArray) {
```

Comment:
🛠 Read the comment below and update your code accordingly:

@jimernest90 while this works, you should use a different parameter for your `forEach()` callback

You're currently using `numArray` as your callback parameter: `.forEach(function(numArray))`. Using this as the same name as your array of numbers (also named `numArray`) is potentially confusing because they are two different variables and hold different values. This makes the logic harder to follow.  

When working with arrays and their iterators (such as`.forEach()`) should adhere to the following rules of thumb:

- Variable names of your arrays **plural**
- Parameter names of your iterators should be **singular** 

For example, if I re-wrote your code using the above rules, I'd have code that looked like the following:

```
let numbers = [9, 18, 62, 3, 17] // <-- changed numArray to numbers
let sum = 0

numbers.forEach(function(number) {  //<-- made the parameter name of the iterator singular
  sum += number
})

console.log(sum / numberslength)
```